### ISSUE TITLE
Specify account and password for administrator at first run of Docker container

### ISSUE THREAD

**Issue by koenvervloesem** [thingsboard/thingsboard#2790]
I have installed a Docker container with ThingsBoard from the `thingsboard/tb-postgres` image but it creates demo accounts and data. How do I prevent this? I especially want to prevent that the server starts with well-known default passwords set up, so is there a way to specify an account and password for the administrator at the first run of the Docker container?

**Comment by Ultrazombie** [comment]
No, you cannot use the custom Postgres password and login in this deployment (thingsboard/tb-postgres).
